marshal.go

```diff
@@ -41,6 +41,12 @@ type Unmarshaler interface {
 	UnmarshalCQL(info TypeInfo, data []byte) error
 }
 
+// TypeHintedValue is a hinted value to power binding non-prepared stmts
+type TypeHintedValue struct {
+	Value interface{}
+	TInfo TypeInfo
```

Comment:
It seems to me that it would be more readable if the field name was not abbreviated. For example `TypeInfo` or `Type`.